### PR TITLE
expression: panic in test mode if builtin function's PbCode unspecified

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

### What problem does this PR solve? 

1. Update the `tipb` to the latest version which makes detect missing PbCode easily.
2. Panic in test mode if builtin function's PbCode unspecified

### What is changed and how it works?

Update go.mod and add a fail point to build expression to Protobuf.

### Check List 

Tests 

 - Unit test
 - Integration test

Release note

 - No need.

### CODE REVIEW

File: expression/expr_to_pb_test.go [hunk 1]
```diff
@@ -30,6 +31,14 @@ import (
 	"github.com/pingcap/tipb/go-tipb"
 )
 
+func init() {
+	fpname := "github.com/pingcap/tidb/expression/PanicIfPbCodeUnspecified"
```
Comment (zimulala) [inline expression/expr_to_pb_test.go]: Why not put it to `TestPanicIfPbCodeUnspecified`?

Comment (lonng) [reply expression/expr_to_pb_test.go]: It's should affect all tests.

**Pull request merged**